CONTRIBUTING.md

```diff
@@ -116,6 +117,35 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+## Merge bot interaction
+
+Maintainers should never merge anything directly into upstream
+branches.  Instead, interact with the [bors-ng bot](https://bors.tech/)
+through PR comments as summarized below. This ensures all upstream
+branches contain commits in a predictable order, and that every commit
+has passed automated testing at some point in the past. A
+[Maintainer portal](https://app.bors.tech/repositories/22803)
+is available, showing all PRs awaiting review and approval.
+
+Common [bors-ng comment commands](https://bors.tech/documentation/):
+
+* `bors ping` - Confirm bot is functioning, it will post a comment in the PR if so.
+* `bors try` - Optional / simulate merging of one (or more) PRs, executing automated
+  tests on the combination, and reporting results.  Good for validating `bors.toml`
+* `bors r+` - Mark the PR has reviewed.  Once a PR receives the required number of
+  approvals (set in `bors.toml`), the PR will go into the merge queue for testing
```

Comment:
what's an approval?  'LGTM', 'bors r+', or something else?

Reply:
Hmmm, ya that's confusing, I will clarify...